Sources/Confidence/FlagEvaluation.swift

```diff
@@ -77,6 +76,9 @@ extension FlagResolution {
                     resolveReason = .stale
                 }
                 if let typedValue = typedValue {
+                    Task {
+                        await flagApplier?.apply(flagName: parsedKey.flag, resolveToken: self.resolveToken)
```

Comment:
This PR surfaced another issue here: with the recent change of behaviour where STALE returns the old context's value rather than the default value, we might want to reflect that in the apply's resolve token. 

In other words, my understanding is that this apply is now reporting to the backend that the non-stale value has been assigned, while this is not true

Reply:
Looking into it further: the resolve token here is the old one, as expected, so no problems here